javascript/ditsmod/package.json

```diff
@@ -16,11 +16,11 @@
   "license": "MIT",
   "dependencies": {
     "@ditsmod/body-parser": "~2.17.1",
-    "@ditsmod/core": "~2.61.0",
+    "@ditsmod/core": "~2.61.1",
     "@ditsmod/routing": "~2.7.0"
   },
   "devDependencies": {
-    "@types/node": "22",
-    "typescript": "5.6.3"
+    "@types/node": "^20.6.1",
```

Comment:
We are using the last LTS of node here, 22.

Reply:
It doesn't affect the runtime though, but I'll update to version 22 now.

Reply:
sure, just a matter of providing an clean approach, for people without node / tech background